integration/test-integration/src/test/resources/apim/1/apis/SOAPApi.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "SOAPApi",
+  "version": "1.0.0",
+  "context": "soap",
+  "type": "SOAP",
+  "tiersCollection": "Unlimited",
+  "wsdlInfo": {
+    "type": "WSDL"
+  },
+  "wsdlUrl": "wsdl/PhoneVerification.wsdl",
```

Comment:
move this to the set 2

Reply:
As we don't need to test this specifically for eventhub scenarios, I think its fine to keep this under set 1 and keep the wsdlurl as the file path because, anyway we will be getting the wsdl file content as a String in either the scenarios.